python/lsst/jointcal/jointcal.py

```diff
@@ -696,21 +730,23 @@ def _check_stars(self, associations):
                 self.log.warn("ccdImage %s has only %s RefStars (desired %s)",
                               ccdImage.getName(), nRefStars, self.config.minRefStarsPerCcd)
 
-    def _iterate_fit(self, associations, fit, model, max_steps, name, whatToFit):
+    def _iterate_fit(self, associations, fit, model, max_steps, name, whatToFit, doRankUpdate=True):
         """Run fit.minimize up to max_steps times, returning the final chi2."""
 
         for i in range(max_steps):
-            r = fit.minimize(whatToFit, 5)  # outlier removal at 5 sigma.
+            # outlier removal at 5 sigma.
+            r = fit.minimize(whatToFit, 5, doRankUpdate=doRankUpdate)
```

Comment:
This is pre-existing, but the sigma threshold for outlier rejection should be configurable.

Reply:
Good point. I've been meaning to do that for a while. I've added `config.outlierRejectSigma` in a separate commit.